lib/tests/streamlit/elements/arrow_altair_test.py

```diff
@@ -71,7 +71,7 @@ def test_altair_chart(self):
             },
         )
         self.assertEqual(spec_dict["data"], {"name": proto.datasets[0].name})
-        self.assertEqual(spec_dict["mark"], "bar")
+        self.assertIn(spec_dict["mark"], ["bar", {"type": "bar"}])
```

Comment:
We check both here `bar` and `{"type": "bar"}` to allow our tests to run with v4 and v5 versions.